index.js

```diff
@@ -1,3 +1,79 @@
-var binding = require('node-gyp-build')(__dirname);
-module.exports = binding;
+class ShortcutHelper {
```

Comment:
Indentations look off, should be 2 spaces per https://github.com/zelloptt/zello-code-style/tree/master/javascript#zello-javascript-code-style-and-standards